internal/controller/controller.go

```diff
@@ -223,36 +223,52 @@ func (c *Controller) startInformers(stopCh <-chan struct{}) {
 }
 
 func (c *Controller) createMeshServices() error {
-	services, err := c.clients.GetServices(metav1.NamespaceAll)
+	potentialSvcs, err := c.clients.GetServicesWithSelectors(
+		metav1.NamespaceAll,
+		c.ignored.LabelSelector(),
+		c.ignored.FieldSelector(),
+	)
 	if err != nil {
 		return fmt.Errorf("unable to get services: %v", err)
 	}
 
-	for _, service := range services {
-		if c.ignored.IsIgnoredService(service.Name, service.Namespace) {
-			continue
-		}
+	maeshSvcs, err := c.clients.GetServicesWithSelectors(c.meshNamespace, "app==maesh", "")
```

Comment:
Could we codify this with

`c.getMeshServiceSelector()` or something like that instead of the "app==maesh" label? I'm thinking something that we could re-use in other packages if needed